db/setup/storage-download/storage-download.py

```diff
@@ -0,0 +1,52 @@
+import os
+from azure.storage.blob import BlobServiceClient
+
+
+def folder_processing(storage_url, container_name):
+    """
+    This function downloads all the files from a container in a storage account
+    to the local directory "test"
+
+    This serves as a way to locally download the container files for processing and importing within the db
+
+    Parameters:
+    - storage_url: the url of the storage account
+    - container_name: the name of the container
+
+    Returns: None
+    """
+
+    # Create a blob service client
+    blob_service_client = BlobServiceClient.from_connection_string(conn_str=storage_url)
+
+    # # Get the container client
+    container_client = blob_service_client.get_container_client(container_name)
+    if container_client.exists():
+        print("Container exists")
+    else:
+        print("Container does not exist")
+    # print("===============================================")
+    # print(container_name)
+    # Specify the local directory
+    local_dir = "test"
+
+    # List blobs in the container
+    blob_list = container_client.list_blobs()
+    i = 0
+    # Iterate through each blob
+    for blob in blob_list:
+        # Create a blob client
+        blob_client = blob_service_client.get_blob_client(
```

Comment:
container_client also has get_blob_client so you don't need to go back to service_client:

https://learn.microsoft.com/en-us/python/api/azure-storage-blob/azure.storage.blob.containerclient?view=azure-python#azure-storage-blob-containerclient-get-blob-client

meaning we can stick to passing the container_client to the function and using that.